package.json

```diff
@@ -22,7 +22,8 @@
     "ts:check": "tsc",
     "test": "mocha -r ts-node/register tests/**/*.test.ts",
     "test:e2e": "mocha -r ts-node/register e2e/tests/**/*.test.ts",
-    "ci": "npm run lint && npm run build && npm run ts:check && npm run test"
+    "ci": "npm run lint && npm run build && npm run ts:check && npm run test",
+    "coverage": "(cp -r src src_tmp && ts-node scripts/insertCoverageTracing.ts && yarn build && yarn test) ; (rm -rf src && mv src_tmp src)"
```

Comment:
it feels a bit over engineered, but I guess this is a hacky way u guys found to add the coverage. Happy if it works